src/hex_http.erl

```diff
@@ -8,14 +8,14 @@
 -type method() :: get | post | put | patch | delete.
 -type status() :: non_neg_integer().
 -type headers() :: #{binary() => binary()}.
--type body() :: nil.
+-type body() :: binary() | {string(), binary()} | nil.
```

Comment:
What is `{string(), binary()}` used for? /cc @wojtekmach

Reply:
Body can be the tuple `{ContentType, Body}`. It is used in `hex_http_httpc`.